pmg/models/users.py

```diff
@@ -113,6 +117,10 @@ class User(db.Model, UserMixin):
     def __unicode__(self):
         return unicode(self.email)
 
+    @property
+    def confirmed(self):
```

Comment:
the to_dict property that this is bringing to the backend was already called `confirmed` and I didn't want to change the API (largely because I'm lazy, but also cos I'm not sure who might be using this). Do you think it's safe to change the API?